.gitignore

```diff
@@ -1,4 +1,20 @@
-
-.DS_Store
+*.tmp
+src/ontology/mirror
+src/ontology/mirror/*
+semantic.cache
+bin/
```

Comment:
Use the file generated by the ODK with seed my repo, this one here is missing a lot of entries.

Reply:
yeah I don't know why it was empty, so i copied stuff over from PCL that made sense - will see if I can grab it from somewhere else

Reply:
copied over stuff from mondon